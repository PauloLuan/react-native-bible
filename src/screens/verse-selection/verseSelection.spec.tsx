import { waitFor } from '@testing-library/react-native'
import React from 'react'
import { renderWithTheme } from '../../utils/test-helper'
import { VerseSelection } from './verseSelection.component'

describe('<VerseSelection />', () => {
  const testId = 'component-verse-selection-id'

  it('should render the VerseSelection component', async () => {
    const { queryByTestId, toJSON } = renderWithTheme(
      <VerseSelection testId={testId} />
    )
    await waitFor(() => expect(queryByTestId(testId)).toBeTruthy())
    expect(toJSON()).toMatchSnapshot()
  })
})
