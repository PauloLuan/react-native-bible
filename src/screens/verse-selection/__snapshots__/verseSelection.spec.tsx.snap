// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerseSelection /> should render the VerseSelection component 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "blue",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "height": 60,
        "width": 85,
      },
    ]
  }
  testID="component-verse-selection-id"
/>
`;
